libs/aws/odc/aws/queue.py

```diff
@@ -16,13 +17,16 @@ def publish_message(queue, message: str, message_attributes: Mapping[str, Any] =
     string.
     """
     queue.send_message(
-        QueueUrl=queue.url,
-        MessageBody=message,
-        MessageAttributes=message_attributes
+        QueueUrl=queue.url, MessageBody=message, MessageAttributes=message_attributes
     )
 
 
-def get_messages(queue, limit: bool = None, visibility_timeout: int = 60, message_attributes: Iterable[str] = ["All"]):
+def get_messages(
+    queue,
+    limit: bool = None,
+    visibility_timeout: int = 60,
+    message_attributes: Iterable[str] = ["All"],
+):
     """
```

Comment:
needs docs, particularly units for timeout

Reply:
Added documentation.